CMakeLists.txt

```diff
@@ -229,6 +233,11 @@ else()
   message(STATUS "FCL does not use OctoMap (as requested)")
 endif()
 
+# Need to include directory to find export file
+include_directories(${PROJECT_BINARY_DIR}) 
+
+install(FILES ${PROJECT_BINARY_DIR}/${PROJECT_NAME}_export.h
```

Comment:
Could we change the export header to simply `export.h`? I don't see any ambiguity or confliction in including this file if we install this file under `${CMAKE_INSTALL_INCLUDEDIR}/fcl`. In this way, we could include this file as:
```cpp
#include "fcl/export.h" // in FCL code base

#include <fcl/export.h> // in FCL dependent projects.
```